state/accumulator.go

```diff
@@ -389,6 +390,7 @@ func (a *Accumulator) Accumulate(txn *sqlx.Tx, userID, roomID string, prevBatch
 				})
 				sentry.CaptureMessage(msg)
 			})
+			// the HS gave us bad data so there's no point retrying
 			// by not returning an error, we are telling the poller it is fine to not retry this request.
 			return 0, nil, nil
```

Comment:
I wonder if this ought to be a data error too?

Reply:
In both cases we log and continue. Unsure either way. Inertia says leaving this.